code/renderers/vue/src/render.ts

```diff
@@ -81,7 +82,7 @@ export const render: ArgsStoryFn<VueRenderer> = (args, context) => {
     .map((argType) => argType.name);
 
   if (eventProps.length) {
-    eventsBinding = eventProps.map((name) => `@${name}="$props.${name}"`).join(' ');
+    eventsBinding = eventProps.map((name) => `@${name}="$props['${name}"']`).join(' ');
```

Comment:
@ndelangen As far as I know Vue2 will convert kebab-case prop to camelCase props. so if you define `also-clicked` in `argTypes` or story `args`, you will see only `alsoClicked` in `props`

<img width="400" alt="Screenshot at Feb 01 22-03-39" src="https://user-images.githubusercontent.com/51897872/216080320-47aa0442-c523-468b-9cce-06ad623e8803.png">
<img width="366" alt="Screenshot at Feb 01 22-03-15" src="https://user-images.githubusercontent.com/51897872/216080343-e5d0f0d0-ef8b-476b-ab60-a3e72d409b69.png">